File: eccorce_app/app/src/main/java/com/example/eccomerce_app/dto/response/BannerResponseDto.kt
package com.example.e_commercompose.dto.response

import com.example.hotel_mobile.services.kSerializeChanger.UUIDKserialize
import kotlinx.serialization.Serializable
import java.util.UUID
@Serializable
data class BannerResponseDto(
    @Serializable(with= UUIDKserialize::class)
    val id: UUID,
    val image:String,
    @Serializable(with= UUIDKserialize::class)
    val store_id: UUID,
)

File: ecommerc_dotnet/dto/Response/BannerResponseDto.cs
namespace ecommerc_dotnet.dto.Response;

public class BannerResponseDto
{
    public Guid id { get; set; }
    
    public String image { get; set; }
    
    public DateTime  endAt{ get; set; }
    public DateTime createdAt { get; set; }

    public Guid storeId { get; set; }
}